components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit-oidc.jsp

```diff
@@ -158,7 +160,11 @@
         tokenUrl = StringUtils.EMPTY;
     }
     if (StringUtils.isBlank(callBackUrl)) {
-        callBackUrl = IdentityUtil.getServerURL(IdentityApplicationConstants.COMMONAUTH, true, true);
+        try {
+            callBackUrl = ServiceURLBuilder.create().addPath(IdentityApplicationConstants.COMMONAUTH).build().getAbsolutePublicURL();
+        } catch(URLBuilderException e) {
+            throw new RuntimeException("Error occurred while building URL in tenant qualified mode.", e);
```

Comment:
We might run into an exception even when it is not the tenant qualified mode. Can we have a generic message so that the users won't be confused in case the mode is not 'tenant-qualified-mode'.

Reply:
fixed with https://github.com/wso2/carbon-identity-framework/pull/3015